include/highfive/bits/H5Slice_traits_misc.hpp

```diff
@@ -71,54 +71,38 @@ inline ElementSet::ElementSet(const std::vector<std::vector<std::size_t>>& eleme
     }
 }
 
+template <typename Derivate>
+inline Selection SliceTraits<Derivate>::select(const HyperSlab& hyperslab) const {
+    const auto& slice = static_cast<const Derivate&>(*this);
+    auto space = hyperslab.apply(slice.getSpace());
+
+    return Selection(space, space, details::get_dataset(slice));
```

Comment:
I think the error is here: before the first arg was `DataSpace(count)` that means nothing here...